.github/CODEOWNERS

```diff
@@ -0,0 +1,2 @@
+# Note: change to suit your needs.
```

Comment:
:eyes:

Reply:
Yes, this comes from my rust-template repo :) - removed.